src/ResourceManager/Reservations/Commands.Reservations/help/Update-AzureRmReservation.md

```diff
@@ -76,6 +76,19 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.```yaml
```

Comment:
@corquiri the build is failing because of the trailing ` ```yaml ` string, please move it to a new line (see other parameters as an example)

Reply:
Moved